core/src/main/java/org/mule/runtime/core/internal/event/AbstractEventContext.java

```diff
@@ -150,14 +150,14 @@ public final void success() {
   @Override
   public final void success(CoreEvent event) {
     if (isResponseDone()) {
-      if (LOGGER.isDebugEnabled()) {
-        LOGGER.debug(this + " response was already completed, ignoring.");
+      if (debugLogEnabled) {
+        LOGGER.debug("{} response was already completed, ignoring.", this);
       }
       return;
     }
 
-    if (LOGGER.isDebugEnabled()) {
-      LOGGER.debug(this + " response completed with result.");
+    if (debugLogEnabled) {
+      LOGGER.debug("{} response completed with result.", this);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: No need to call "toString()" method as formatting and string conversion is done by the Formatter. ([squid:S3457](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7701&issues=AWkLjRWXotIXnmsUmgk9&open=AWkLjRWXotIXnmsUmgk9)

[ISSUE_KEY]: AWkLjRWXotIXnmsUmgk9